after/syntax/python.vim

```diff
@@ -0,0 +1 @@
+syn include syntax/css/vim-coloresque.vim
```

Comment:
Please don't randomly add new file types to this. People can define these if they want to.